Login.xaml.cs

```diff
@@ -0,0 +1,258 @@
+            {
+                Directory.CreateDirectory(pathDir);
+            }
+
+            //string path = Directory.GetParent(System.Reflection.Assembly.GetExecutingAssembly().Location).FullName;
+            //string path = System.IO.Path.Combine(Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName + "\\Credential", "LoginCredential.xml");
+            //string fileName = System.IO.Path.Combine(path, "LoginCredential.xml");
+            string path = AppDomain.CurrentDomain.BaseDirectory + "\\Credential\\LoginCredential.xml";
+
+            try
+            {
+                var Status_DefaultPasswordExist = CheckDefaultPasswordExist();
+
+                if (File.Exists(path))
+                {
+                    if (Status_DefaultPasswordExist == true)
+                    { }
+                }
+                else
+                {
+                    XmlWriterSettings xmlWriterSettings = new XmlWriterSettings();
+                    xmlWriterSettings.Indent = true;
+                    xmlWriterSettings.NewLineOnAttributes = true;
+                    using (XmlWriter xmlWriter = XmlWriter.Create(path, xmlWriterSettings))
+                    {
+                        xmlWriter.WriteStartDocument();
+                        /*LoginCredential Config*/
+                        xmlWriter.WriteStartElement("LoginCredential");
+                        xmlWriter.WriteElementString("UserName", Helper.Crypto.Encrypt(username));
+                        xmlWriter.WriteElementString("Password", Helper.Crypto.Encrypt(password));
+                        xmlWriter.WriteEndElement();
+
+                        xmlWriter.WriteEndDocument();
+                        xmlWriter.Flush();
+                        xmlWriter.Close();
+                    }
+                }
+            }
+            catch(Exception ex)
+            {                
+                SentrySdk.CaptureMessage(ex.Message);
+            } 
+        }
+
+        private bool CheckDefaultPasswordExist()
+        {
+            //string path = Directory.GetParent(System.Reflection.Assembly.GetExecutingAssembly().Location).FullName;
+            //string path = System.IO.Path.Combine(Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName + "\\Credential", "LoginCredential.xml");
+            //string fileName = System.IO.Path.Combine(path, "LoginCredential.xml");
+            string path = AppDomain.CurrentDomain.BaseDirectory + "\\Credential\\LoginCredential.xml";
+
+            DataSet ds = new DataSet();
+            try
+            {
+                ds.ReadXml(path);
+
+                var dt = ds.Tables["LoginCredential"];
+                if (dt != null)
+                {
+                    var getUserName_XML = dt.Rows[0][0].ToString();
+                    var getPass_XML = dt.Rows[0][1].ToString();
+                    var getUserName_Decrypt = Helper.Crypto.Decrypt(getUserName_XML);
```

Comment:
Again, you're not using the `username`, why are you decrypting it?

Reply:
yes you are right .
there is no use of this function .
referenced with the above issue